data/net/minecraft/util/profiling/metrics/profiling/ProfilerSamplerAdapter.mapping

```diff
@@ -1,4 +1,7 @@
 CLASS net/minecraft/util/profiling/metrics/profiling/ProfilerSamplerAdapter
+	METHOD newSamplersFoundInProfiler (Ljava/util/function/Supplier;)Ljava/util/Set;
+		ARG 1 collecterSupplier
 	METHOD samplerForProfilingPath (Ljava/util/function/Supplier;Ljava/lang/String;Lnet/minecraft/util/profiling/metrics/MetricCategory;)Lnet/minecraft/util/profiling/metrics/MetricSampler;
+		ARG 0 collectorSupplier
```

Comment:
```suggestion
		ARG 1 profiles
	METHOD samplerForProfilingPath (Ljava/util/function/Supplier;Ljava/lang/String;Lnet/minecraft/util/profiling/metrics/MetricCategory;)Lnet/minecraft/util/profiling/metrics/MetricSampler;
		ARG 0 profiles
```

I really don't like this name as is, although I do see the difficulty of naming it correctly. This is consistent with the same parameter usage in [`MetricsSampleProvider`](https://github.com/ParchmentMC/Parchment/blob/b44afe5f2b885025f8f41653dffbbec5db7afa48/data/net/minecraft/util/profiling/metrics/MetricsSamplerProvider.mapping#L3) which is ultimately passed in here.